pkg/kf/commands/config/config.go

```diff
@@ -277,8 +275,15 @@ func getRestConfig(p *KfParams) *rest.Config {
 	return restCfg
 }
 
-func initKubeConfig(p *KfParams) {
-	if p.KubeCfgFile == "" {
-		p.KubeCfgFile = filepath.Join(homedir.HomeDir(), ".kube", "config")
+func initKubeConfig() KfParams {
+	p := KfParams{}
+
+	p.KubeCfgFile = filepath.Join(homedir.HomeDir(), ".kube", "config")
+
+	// Override path to Kube config if env var is set
+	if configPathEnv, ok := os.LookupEnv("KUBECONFIG"); ok {
```

Comment:
Instead of hardcoding "KUBECONFIG", could we use the constant `RecommendedConfigPathEnvVar` provided in https://godoc.org/k8s.io/client-go/tools/clientcmd#pkg-constants